### ISSUE TITLE
TabMigrate shows hidden sheets after migrate 

### ISSUE THREAD

**Issue by ghost** [tableau/TabMigrate#54]
Not sure if this has been reported before so here it goes:
When publishing a Workbook with multiple sheets and Dashbaords, TabMigrate shows all the content.
I have experimented with a Workbook that is made up of multiple sheets but originally only Dashbords are visible.
When i migrate it to another server, TabMigrate shows all the sheets

**Comment by Pceaton** [comment]
Yes, I have just noticed this same issue while testing the tool.

**Comment by abhi234u** [comment]
Will there be a fix to the hidden sheets? Thanks
